WalletWasabi.Tests/UnitTests/UserInterfaceTest/PocketSelectionTests.cs

```diff
@@ -377,26 +377,6 @@ public async Task DoNotUsePrivateFundsAsync()
 		Assert.DoesNotContain(pocket3, output);
 	}
 
-	[Fact]
-	public async Task IncludePrivateFundsAsync()
```

Comment:
This can be safely removed as in this scenario semi private needs to be included too.
So `IncludePrivateAndSemiPrivateFundsAsync` actually covers that case.